File: demos/05-maps/maps.ts
const scores = new Map<string, number>([["Go", 0]]);
scores.set("TS", 2);
scores.set("TS", (scores.get("TS") ?? 0) + 1);
console.log(scores.get("Go"), scores.get("Rust"), scores.size);
console.log(scores.get("Go"), scores.has("Go"));
console.log(scores.get("Rust"), scores.has("Rust"));
scores.delete("TS");
console.log(scores.size);

File: demos/05-maps/maps.go
package main

import "fmt"

func main() {
	scores := map[string]int{"Go": 0}
	scores["TS"] = 2
	scores["TS"]++
	fmt.Println(scores["Go"], scores["Rust"], len(scores))
	value, ok := scores["Go"]
	fmt.Println(value, ok)
	value, ok = scores["Rust"]
	fmt.Println(value, ok)
	delete(scores, "TS")
	fmt.Println(len(scores))
}
